Tests/scripts/infrastructure_tests/test_configure_tests.py

```diff
@@ -16,7 +16,8 @@ class TestChangedPlaybook:
     def test_changed_runnable_test__unmocked_get_modified_files(self):
         filterd_tests = get_mock_test_list(git_diff_ret=self.GIT_DIFF_RET)
 
-        assert filterd_tests == {self.TEST_ID}
+        assert filterd_tests[0] == {self.TEST_ID}
+        assert filterd_tests[1] == set()
```

Comment:
[1] is basicaly the pack? if so please split filterd_tests to be two args

Reply:
yea as i noted above, tests are still in the works